components/GithubButton.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import Link from "next/link";
+
+interface GithubButtonProps {
+  text?: string;
+  href?: string;
+}
+
+const GithubButton: React.FC<GithubButtonProps> = ({
+  text = "Star on Github",
+  href = "https://github.com/json-schema-org",
+}) => {
+  return (
+    <div className="inline-block">
+      <Link href={href}>
+        <button className="flex items-center bg-blue-500 hover:bg-blue-600 text-white font-semibold text-sm py-2 px-4 rounded-full transition duration-300 ">
```

Comment:
Let's avoid the blue background colour. I'll look better keeping white color for background, with black letter and some slight mouse hover effect.